### ISSUE TITLE
Use Robocopy instead of Copy

### ISSUE THREAD

**Issue by Ranman86** [JohnnyFFM/XPlotterGui#1]
Is it possible to use robocopy (Microsoft Tool) instead of copy? This will allow copying to Networkpaths ;-)

**Comment by JohnnyFFM** [comment]
Yes it should be possible, will have a look when I have some time 😁

**Comment by JohnnyFFM** [comment]
Hi, 
compiled a version that uses robocopy instead of copy. Would need someone to test it.
Only works with PoC2, removed PoC1 support (needs Xplotter v.1.31 or higher). 

[XPlotter_GUI_robocopy.zip](https://github.com/JohnnyFFM/XPlotterGui/files/2134152/XPlotter_GUI_robocopy.zip)
